### PR TITLE
Controllers

### PR DESCRIPTION
**Pull request by kiboma2021**
- Use controllers to handle requests and render empty views.
- Understand Rails naming conventions.
- Use params from browser requests in a safe way.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class PostsController < ApplicationController
+end
```
Comment (KDlamini) [inline app/controllers/posts_controller.rb]: Kindly add the `index` and `show`  actions in your post controller. You can leave them empty for now.

For example:
```
class PostsController < ApplicationController
  
  def index; end

  def show; end

end
```

* Remember that the `index` and `show` methods correspond with the `app/views/posts/index.html.erb` and `app/views/posts/show.html.erb` files.

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+class UsersController < ApplicationController
+  def likes; end
+
+  def posts; end
+
+  def comments; end
+end
```
Comment (KDlamini) [inline app/controllers/users_controller.rb]: Rails conventions require you to use the `new`, `index`, `show`, `create`, and `destroy` actions/methods to add your [CRUD](https://www.sumologic.com/glossary/crud/) functionalities.

**new** - for initiating a new user
**index** - for showing all users from the database to your application
**show** - for showing only one specific user given the `user_id`
**create** - for creating a new user and saving it to the database
**destroy** - for deleting a user permanently in the database

- Kindly add the `index` and `show` methods as demonstrated in the comment above.

This is an example of an implementation of CRUD (Create, Read, Update, Delete) using rails conventions for `posts_controller`. You will implement something similar in the future.

Screenshot from 2022-10-25 20-49-55

- In Rails, the methods will always be named as I explained and illustrated above in all your controllers... ALWAYS.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @kiboma2021 🙋‍♀️,  

Great work so far! 💯 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞 

✔️  Correct PR title and description
✔️  Well done adding the proper `routes` and `views`
✔️  No linter errors

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/users_controller.rb (hunk 2 above)
Comment (kiboma2021) [reply app/controllers/users_controller.rb]: @KDlamini , This has now been implemented

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @kiboma2021👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**